cli/flox-rust-sdk/src/flox.rs

```diff
@@ -225,6 +225,30 @@ pub mod test_helpers {
     use super::*;
     use crate::providers::git::{GitCommandProvider, GitProvider};
 
+    pub fn create_test_token(handle: &str) -> Result<FloxhubToken, FloxhubTokenError> {
```

Comment:
thanks for adding this!

 
NB: we already use a (hardcoded) fake token in some places, could you replace those with this function perhaps?

Reply:
Probably?  Added a note to follow up.